storage/src/robocorp/storage/_storage.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from ._requests import Requests
+from ._utils import required_env, url_join
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_assets_client():
+    """Creates and returns an API client based on the injected env vars in CR."""
+    api_url = required_env("RC_API_URL_V1")
+    api_token = required_env("RC_API_TOKEN_V1", "")
+    api_key = required_env("RC_API_KEY", "")  # for local tests only
```

Comment:
@sahokas Do you think this can pose any security risk? (introducing the ability to make requests with env var injected API keys -- not designed for production, but makes local testing easier)